projects/chef-infra.md

```diff
@@ -37,8 +37,10 @@ Mark Anderson
 ### Project Repositories
 
 - [bft](https://github.com/chef/bft)
+- [chef-api](https://github.com/chef/chef-api)
+- [chef-ruby-lvm-attrib](https://github.com/chef-ruby-lvm-attrib)
```

Comment:
missing the chef org here i think in the url.

Reply:
fixed